src/test/java/kitchenpos/application/fixture/ProductFixture.java

```diff
@@ -0,0 +1,20 @@
+package kitchenpos.application.fixture;
+
+import java.math.BigDecimal;
+import kitchenpos.domain.Product;
+
+public class ProductFixture {
+
+    private ProductFixture() {
+    }
+
+    public static Product 상품생성(Long id, String name, Integer price) {
```

Comment:
픽스처를 잘 작성해주셨네요~

```suggestion
    public static Product 양념치킨() {
    ...
    ...
    return product;
    }
```

이런식으로 직접 값을 지정해주고 메서드 명으로 객체를 정의해주는 방식의 픽스처도 사용해볼 수 있겠네요 :)
두 방법중 편하신걸로 사용하시면 좋겠습니다 :)